gst-libs/gst/pbutils/codec-utils.c

```diff
@@ -67,23 +72,39 @@ digit_to_string (guint digit)
  * rate.
  *
  * Returns: The sample rate if @sr_idx is valid, 0 otherwise.
- *
- * Since: 0.10.31
```

Comment:
why do you remove this ?

Reply:
No reason. I merged the code from 1.0 and that comment was accidentally removed because it was also removed from 1.0.